src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { InputHTMLAttributes, ReactNode, useState } from 'react';
+import { Typography } from '..';
+import cx from 'classnames';
+import styles from './Checkbox.module.scss';
+import { Colors } from '../../styles';
+
+export interface CheckboxProps extends InputHTMLAttributes<HTMLInputElement> {
+  label?: ReactNode;
+  disabled?: boolean;
+  checked?: boolean;
+  color?: Colors;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => unknown;
+}
+
+function Checkbox({
+  checked,
+  onChange,
+  color = 'primary',
+  label,
+  children,
+  ...props
+}: CheckboxProps) {
+  const [_checked, setChecked] = useState<boolean>(Boolean(checked));
+
+  function _onChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setChecked(event.target.checked);
+    onChange?.(event);
+  }
```

Comment:
Why not just use the props directly? Why are we proxying it?

Reply:
Needed it for the checked class for the fake checkbox; I could change the selector to `input:checked + .checkbox` though